addon/components/zoom-zone.js

```diff
@@ -53,18 +53,15 @@ export default Component.extend({
   },
 
   willDestroyElement() {
-    const viewport = this.get('$viewport');
-    const content = this.get('$content');
-
-    viewport.off();
-    content.off();
+    this.get('$viewport').off();
+    this.get('$content').off();
 
     this._super(...arguments);
   },
 
   matrix: computed('panX', 'panY', 'scale', function () {
-    const [scale, x, y] = [this.get('scale'), this.get('panX'), this.get('panY')];
-    return `matrix(${scale}, 0, 0, ${scale}, ${x}, ${y})`;
+    const { scale, panX, panY } = this;
```

Comment:
I'm not certain accessing these directly won't cause issues if other code uses observers on them. Maybe use `getProperties`?